forward_engineering/alterScript/alterScriptHelpers/alterEntityHelper.js

```diff
@@ -132,7 +132,7 @@ const getAddColumnScriptDtos =
  * */
 const getIndexesBasedOnNewlyCreatedColumnsScript = ({_, ddlProvider, dbVersion, collection}) => {
     const newIndexes = collection?.role?.compMod?.Indxs?.new || collection?.role?.Indxs || []
-    const newPropertiesIds = Object.values(collection?.properties).map(({GUID}) => GUID)
+    const newPropertiesIds = Object.values(collection?.properties ?? {}).map(({GUID}) => GUID)
```

Comment:
Can we do a preemptive exit from the function? Looking at the if statement below, and your _nullish_ check, it seems that we can do the following on the first lines of this function:
```
if (!collection?.properties) {
  return []
}
```